packages/ens-utils/tsup.config.ts

```diff
@@ -2,9 +2,10 @@ import { defineConfig } from "tsup";
 
 export default defineConfig({
   entry: ["src/index.ts"],
-  splitting: true,
-  sourcemap: false,
+  splitting: false,
+  sourcemap: true,
   clean: true,
+  bundle: true,
   dts: true,
   treeshake: true,
   target: [
```

Comment:
I notice a few distinct configurations here for `ens-utils` but not other packages. Is there any special reason all packages don't use the same config?

1. `target` (all our packages are supposed to run in node and in browsers, not only ens-utils)
2. `skipNodeModulesBundle`

Additionally, I see we have `external` set to `["node_modules"]` here. I assumed we wouldn't need to specify anything explicit as `external`? As I understand any dependencies in package.json are automatically excluded? https://tsup.egoist.dev/#excluding-packages